modules/web/src/plugins/debugger/plugin.js

```diff
@@ -110,16 +110,16 @@ class DebuggerPlugin extends Plugin {
                     region: REGIONS.LEFT_PANEL,
                     // region specific options for left-panel views
                     regionOptions: {
-                        activityBarIcon: 'start',
-                        panelTitle: 'Run',
+                        activityBarIcon: 'bug',
+                        panelTitle: 'Debugger',
                         panelActions: [
                             {
-                                icon: 'configarations',
+                                icon: 'configurations',
```

Comment:
This is intensionally misspelled because it is a typo in font wso2. It is corrected in the font wso2 repo now. We should correct this when we get font Wso2 latest

Reply:
Since we build our font now we should just rename this file https://github.com/ballerinalang/composer/blob/master/modules/web/font/font-ballerina/icons/configarations.svg

@dakshika could you please confirm?

Reply:
I have renamed the svg file as well with this commit. So this should work